gui/default/syncthing/core/durationFilter.js

```diff
@@ -21,7 +21,11 @@ angular.module('syncthing.core')
                 var t = (input / SECONDS_IN[k] | 0); // Math.floor
 
                 if (t > 0) {
-                    result += " " + t + k;
```

Comment:
Yep, this script is currently used to display uptime and rescan interval, and in both cases the only purpose of the spaces seems to be to separate different values, so that we end up with `1d 2h 30m` and not `1d2h30m`. In the former, the leading space is actually trimmed by the browser anyway, and in the latter, it only adds unnecessary spacing between the number and the icon. The additional space is unnecessary, because there is already an `&nbsp;` there.